### PR TITLE
v3.39.0

### PR DESCRIPTION
**Pull request by TheTrunk**

### CODE REVIEW

**Review by Cabecinha84 — CHANGES_REQUESTED** [review]
@TheTrunk can we change where we have = this.originalInstances; to  = this.originalInstances();
It's a method and I think it's more easy to read.

**Review by Cabecinha84 — APPROVED** [review]
Commited a fix for locked instances. All tested now and working good.
HomeUi needs to be rebuild before release.
Lets schedule to release this on 21st 10am UTC if it's ok for you @TheTrunk 
When Ann lets mark as a mandatory version because of the upnpbench fix.

File: ZelBack/src/services/benchmarkService.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ async function executeUpnpBench() {
   const isUPNP = upnpService.isUPNP();
   if ((userconfig.initial.apiport && userconfig.initial.apiport !== config.server.apiport) || isUPNP) {
     log.info('Calling FluxBench startMultiPortBench');
-    startMultiPortBench();
+    log.info(await startMultiPortBench());
```
Comment (TheTrunk) [inline ZelBack/src/services/benchmarkService.js:252]: That is pretty strange @Cabecinha84

Comment (Cabecinha84) [reply ZelBack/src/services/benchmarkService.js:252]: what you get:
2023-04-20T16:04:05.740Z          Calling FluxBench startMultiPortBench
2023-04-20T16:04:05.742Z          {"status":"success","data":"Multiport Benchmarks will start in the next couple minutes"}

**Comment by Cabecinha84** [comment]
> @TheTrunk can we change where we have = this.originalInstances; to = this.originalInstances();
> It's a method and I think it's more easy to read.

@TheTrunk don't forget this one.

**Pull request merged**